src/bitcoin/utilities/accounts/AccountFactory.ts

```diff
@@ -20,7 +20,7 @@ export function generateAccount(
     instanceNum: number,
     accountName: string,
     accountDescription: string,
-    primarySeed: string,
+    primarySeed?: string,
```

Comment:
The primary seed can not be optionalized, it is a must-pass parameter.

Reply:
Resolved